pipe-cli/src/model/pipeline_run_parameter_model.py

```diff
@@ -13,10 +13,27 @@
 # limitations under the License.
 
 class PipelineRunParameterModel(object):
-    def __init__(self, name, value, parameter_type, required):
+    def __init__(self, name, value, parameter_type, required, roles=set()):
         self.name = name
         self.value = value
         self.parameter_type = parameter_type
         if parameter_type is None:
             self.parameter_type = 'string'
         self.required = required
+        self.roles = set() if roles is None else set(roles)
+
+    @staticmethod
+    def load_from_default_system_parameter(parameter_json):
+        parameter_name = None
+        parameter_value = None
+        parameter_type = None
+        parameter_roles = []
+        if 'name' in parameter_json:
+            parameter_name = parameter_json['name']
```

Comment:
Same here.

```suggestion
        parameter_name = parameter_json.get('name')
```

Reply:
Done, applied it to the extraction of other details as well